stix2/base.py

```diff
@@ -220,13 +247,16 @@ def __deepcopy__(self, memo):
         if isinstance(self, _Observable):
             # Assume: valid references in the original object are still valid in the new version
             new_inner['_valid_refs'] = {'*': '*'}
-        new_inner['allow_custom'] = self._allow_custom
-        return cls(**new_inner)
+        return cls(allow_custom=True, **new_inner)
```

Comment:
Would this be?

```suggestion
        return cls(allow_custom=self._allow_custom, **new_inner)
```

Reply:
Remember one of the key changes is that there is no longer an`_allow_custom` attribute maintained on stix2 objects.  So the suggestion can't work.  We could use `self.has_custom`, but it wouldn't do anything different.  If `self` has customizations, we want to allow them here; if `self` doesn't have customizations, allowing them doesn't matter (the implementation will notice there are no customizations and set the `has_custom` property to False).  So `allow_custom=True` is simple and gives us the correct behavior in both cases.  I guess one could argue though, that using `self.has_custom` *looks* better.

Reply:
Ahh I see! No change here then.